cmd/notifiers.go

```diff
@@ -61,6 +63,7 @@ type notifier struct {
 	Redis         map[string]redisNotify         `json:"redis"`
 	PostgreSQL    map[string]postgreSQLNotify    `json:"postgresql"`
 	Kafka         map[string]kafkaNotify         `json:"kafka"`
+	HTTP          map[string]httpNotify          `json:"http"`
```

Comment:
Add a new entry to config requires us to do config migration i.e update the config version and read the code in `config-mgirate.go` which needs to migrate the current config to newer one. 

It might be confusing - i can send you a PR with those changes if needed.